src/main/groovy/com/deploygate/gradle/plugins/DeployGatePlugin.groovy

```diff
@@ -54,108 +55,88 @@ class DeployGatePlugin implements Plugin<Project> {
 
         GradleCompat.init(project)
 
-        project.tasks.register(Constants.LOGIN_TASK_NAME, LoginTask) { task ->
-            task.description = "Check the configured credentials and launch the authentication flow if they are not enough."
+        def loginTask = project.tasks.register(Constants.LOGIN_TASK_NAME, LoginTask) { task ->
+            DeployGateExtension deploygate = project.deploygate
 
-            task.group = Constants.TASK_GROUP_NAME
-            task.deployGateExtension = project.deploygate
+            task.deployGateExtension = deploygate
+            task.appOwnerName.set(deploygate.appOwnerName)
```

Comment:
To use Gradle's caching system, here should pass appOwnerName and apiToken as inputs instead of extension delegation.